monai/bundle/properties.py

```diff
@@ -216,3 +221,42 @@
         BundlePropertyConfig.REF_ID: f"evaluator{ID_SEP_KEY}key_val_metric",
     },
 }
+
+MetaProterties = {
+    "version": {
+        BundleProperty.DESC: "bundle version",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"_meta_{ID_SEP_KEY}version",
+    },
+    "monai_version": {
+        BundleProperty.DESC: "required monai version used for bundle",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"_meta_{ID_SEP_KEY}monai_version",
+    },
+    "pytorch_version": {
+        BundleProperty.DESC: "required pytorch version used for bundle",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"_meta_{ID_SEP_KEY}pytorch_version",
+    },
+    "numpy_version": {
+        BundleProperty.DESC: "required numpy version used for bundle",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"_meta_{ID_SEP_KEY}numpy_version",
+    },
+    "description": {
+        BundleProperty.DESC: "description for bundle",
+        BundleProperty.REQUIRED: False,
+        BundlePropertyConfig.ID: f"_meta_{ID_SEP_KEY}description",
+    },
+    "spatial_shape": {
```

Comment:
Some bundle may have multiple input data? Generative AI bundle may not have input Tensor?

Thanks.

Reply:
Does this cause problems if required is False? My main reference is that MONAILabel needs to get this parameter from the bundle.
https://github.com/Project-MONAI/MONAILabel/blob/43e9e2a982ecac32a34d36b5b9c620b658bb68de/monailabel/tasks/infer/bundle.py#L137

Reply:
OK, it can not exist for generative AI bundles.

Thanks.